### DISCUSSION TITLE
v2021.07.17

### DISCUSSION THREAD

**Discussion by slawekjaranowski** [s4u/pgp-keys-map#296] [Announcements]
# What's Changed

## New features

- Refined keys for groupId "javax.ws.rs" #187
- news keys for org.webjars[.npm] #189
- Added key for more org.sonatype #188
- new key for com.yworks #190
- new key for dnsjava:dnsjava #191
- new key for com.stripe:stripe-java #192
- new key for com.sun.activation #193
- new key for org.apache.tomcat.* #194
- new key for org.libreoffice #195
- new key for io.reforma.semper #198
- new key for jakarta.transaction #199
- Additional keys for net.java.dev.jna #204
- new key for com.google.protobuf:protobuf-java #203
- new key for org.lucee:xalan[-serializer] #196
- new key for com.zaxxer:SparseBitSet #197
- new key for mysql:mysql-connector-java #202
- new key for com.sun.mail:javax.mail #205
- new key for javax.xml.rpc #206
- new key for org.apache.axis2 #207
- new key for org.apache.taglibs #208
- new key for org.swinglabs.swingx #209
- Additional key for junit #212
- new key for xalan #214
- new key for org.apache.pdfbox #211
- Additional key for org.apache.xmlgraphics #213
- new key for de.rototor.pdfbox #210
- new key for org.postgresql:postgresql #215
- new keysmore specific "noSig" for org.antlr #216
- new key for org.apache.commons:commons-math3 #217
- new key for org.eclipse.jdt.core.compiler #219
- new key for org.apache.poiorg.apache.xmlbeans:xmlbeans #220
- new key for org.apache.tomcat.maven #218
- new key for org.jfree:jfreechart #221
- new keys for Jakarta JSTL #224
- new key for org.apache.axis #225
- new key for org.codehaus.mojo:properties-maven-plugin #226
- new key for org.eclipse.jdt:ecj #223
- new keys for org.lz4 #229
- new key for com.hynnet:jacob #230
- new key for com.github.virtuald #232
- Additional key for org.apache.httpcomponents #233
- new key for org.apache.santuario #235
- new keys for Jetty-related artifacts #222
- new key for org.tmatesoftrelated artifacts signed by same key #228
- new key for com.google.errorprone:error_prone_annotations #234
- new key for com.google.code.maven-scm-provider-svnjava #231
- new key for org.codehaus.mojo:buildnumber-maven-plugin #227
- Changed test to artifact that will stay at "com.aoindustries" #251
- new keysnoSig for log4j:log4j:1.2.15-1.2.16 #254
- Additional key for javax.xml.bind #255
- More precisely mapped noSigadded key for com.sun.xml.bind #256
- noSig for jalopy #257
- new keys for org.apache.james #258
- Thoroughly mapped noSignew keys for org.apache.geronimo.specs #260
- new key for org.apache.xmlbeans:*:2.6.0 #261
- new noSigkey for commons-fileupload #262
- Refined noSignew key for wsdl4j #263
- new key for com.sun.mail:jakarta.mail #264
- new key for org.apache.woden #265
- New key for org.codehaus.woodstox:woodstox-core-asl #267
- new key for org.apache.neethi #259
- Additional key for org.apache.ws.xmlschema #266
- Additional key for com.sun.xml.ws #268
- new keys for org.apache.ws.commons.axiom #269
- Key for org.projectlombok #273
- noSig for com.thoughtworks.qdox <= 1.12 #274
- Additional keys for org.apache.xbean #276
- new keys for org.codehaus.gmaven.*related dependencies #275
- noSig for com.servlets #279
- new key for axis2-1.8.0 #287
- new keys for javax.transaction #288
- additional "noSig"key for commons-cli #289
- new key for org.jibx #290
- new keys for org.apache.xmlbeans:xmlbeans:[3.0.0,3.1.0] #291
- new key for com.google.googlejavaformat #292

## Maintenance

- Fixed sorting of keys from PR #217 #236
- Skip verification on deploy #272

## Dependency updates

- Bump commons-io from 2.10.0 to 2.11.0 in /pgp-keys-map-test1 #200
- Bump jsr311-api from 1.1-ea to 1.1.1 in /pgp-keys-map-test1 #201
- Bump spring-core.version from 5.3.8 to 5.3.9 in /org-springframework-test #237
- Bump stripe-java from 20.63.1 to 20.64.0 in /pgp-keys-map-test1 #238
- Bump pgpverify-maven-plugin from 1.14.0 to 1.14.1 #240
- Bump nmw-oss-parent from 1.13.0 to 1.14.0 in /pgp-keys-map-test1 #241
- Bump pragmatickm-parent from 1.10.2 to 1.11.0 in /pgp-keys-map-test1 #242
- Bump semanticcms-parent from 1.12.0 to 1.13.0 in /pgp-keys-map-test1 #243
- Bump ao-oss-parent from 1.16.1 to 1.17.0 in /pgp-keys-map-test1 #244
- Bump mysql-connector-java from 8.0.25 to 8.0.26 in /pgp-keys-map-test1 #245
- Bump slf4j-api from 1.7.31 to 1.7.32 in /pgp-keys-map-test1 #246
- Bump stripe-java from 20.64.0 to 20.65.0 in /pgp-keys-map-test1 #247
- Bump spring-boot.version from 2.5.2 to 2.5.3 in /org-springframework-test #248
- Bump libreoffice from 7.1.4 to 7.1.5 in /pgp-keys-map-test1 #249
- Bump stripe-java from 20.65.0 to 20.66.0 in /pgp-keys-map-test1 #250
- Bump error_prone_annotations from 2.7.1 to 2.8.0 in /pgp-keys-map-test1 #252
- Bump stripe-java from 20.66.0 to 20.67.0 in /pgp-keys-map-test1 #253
- Bump plexus-utils from 3.3.0 to 3.4.0 in /pgp-keys-map-test1 #270
- Bump plexus-utils from 3.3.0 to 3.4.0 in /pgp-keys-map-test1 #271
- Bump maven-enforcer-plugin from 3.0.0-M3 to 3.0.0 in /org-apache-maven-plugins-test #277
- Bump dnsjava from 3.4.0 to 3.4.1 in /pgp-keys-map-test1 #278
- Bump jaxb-impl from 3.0.2-b01 to 3.0.2 in /pgp-keys-map-test1 #280
- Bump jaxws-tools from 3.0.1 to 3.0.2 in /pgp-keys-map-test1 #281
- Bump error_prone_annotations from 2.8.0 to 2.8.1 in /pgp-keys-map-test1 #282
- Bump stripe-java from 20.67.0 to 20.68.0 in /pgp-keys-map-test1 #283
- Bump tomcat-api from 10.1.0-M2 to 10.1.0-M4 in /pgp-keys-map-test1 #284
- Bump tomcat-embed-core from 10.1.0-M2 to 10.1.0-M4 in /pgp-keys-map-test1 #286
- Bump maven-scm-plugin from 1.11.2 to 1.11.3 in /org-apache-maven-plugins-test #293
- Bump stripe-java from 20.68.0 to 20.69.0 in /pgp-keys-map-test1 #294
- Bump maven-core from 3.8.1 to 3.8.2 in /pgp-keys-map-test1 #295

# Thanks
Many thanks for collaboration on this release for: @aoindustries,   @slawekjaranowski 

<hr /><em>This discussion was created from the release <a href='https://github.com/s4u/pgp-keys-map/releases/tag/v2021.07.17'>v2021.07.17</a>.</em>

**Comment by slawekjaranowski** [comment]
ups should have version `2021.08.17` ... but i will keep it ... the version number is greater as last one ...

**Reply by ao-apps** [reply]
Thank you for the release.
